### PR TITLE
JST eclipse test plugin for future JST integration tests.

### PR DESCRIPTION
**Pull request by psrna**

### CODE REVIEW

**Comment by psrna** [comment]
Retest this please.

**Comment by psrna** [comment]
Retest this please.

**Comment by vpakan** [comment]
* add new plugin to some category to have it available on update site
* consider creating SmokeSuite.java class containing smoke tests

**Comment by psrna** [comment]
Retest this please.

File: tests/org.jboss.tools.jst.ui.bot.test/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: JBoss Tools JST Bower Integration Tests
```
Comment (vpakan) [inline tests/org.jboss.tools.jst.ui.bot.test/META-INF/MANIFEST.MF]: Remove "Bower" string

**Comment by vpakan** [comment]
Add disclaimer to all new java classes

**Pull request merged**